solidity-its/src/main/java/org/sonarsource/solidity/its/SolidityRulingIts.java

```diff
@@ -53,4 +54,10 @@ private void reportIssue(int line, String projectDir) {
       LOG.debug(e.getMessage(), e);
     }
   }
+
+  @Override
+  public void addIssue(ParserRuleContext ctx, String reportMessage, String externalRuleKey) {
+    // TODO Auto-generated method stub
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=63&issues=AWSsvO_Cl6vpliCet4fX&open=AWSsvO_Cl6vpliCet4fX)

[ISSUE_KEY]: AWSsvO_Cl6vpliCet4fX